scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+  #         /etc/kat/rocky.conf
+  #         /etc/kat/bytes.conf
+  #         /etc/kat/boefjes.conf
+  #         /etc/kat/octopoes.conf
+  # Or use this command to do it for you:
+  # sudo sed -i "s|QUEUE_URI= *\$|QUEUE_URI=amqp://kat:${rabbitmq_pass}@127.0.0.1:5672/kat|" /etc/kat/*.conf
+  - name: Update QUEUE_URI setting in rocky.conf, bytes.conf, boefjes.conf and octopoes.conf
+    lineinfile:
+      path: "/etc/kat/{{ item }}"
+      regexp: "QUEUE_URI=.*"
+      line:  "QUEUE_URI=amqp://kat:{{ kat_mq_password }}@127.0.0.1:5672/kat"
+    loop:
+      - rocky.conf
+      - bytes.conf
+      - boefjes.conf
+      - octopoes.conf
+
+  # Configure Bytes credentials
+  # copy the value of BYTES_PASSWORD in /etc/kat/bytes.conf to the setting with the same name in the following files:
+  #     /etc/kat/rocky.conf
+  #     /etc/kat/boefjes.conf
+  #     /etc/kat/mula.conf
+  # This oneliner will do it for you, executed as root:
+  # sudo sed -i "s/BYTES_PASSWORD= *\$/BYTES_PASSWORD=$(grep BYTES_PASSWORD /etc/kat/bytes.conf | awk -F'=' '{ print $2 }')/" /etc/kat/*.conf
+  - name: Update BYTES_PASSWORD setting in rocky.conf, boefjes.conf and mula.conf
+    lineinfile:
+      path: "/etc/kat/{{ item }}"
+      regexp: "BYTES_PASSWORD=.*"
+      line:  "BYTES_PASSWORD={{ bytes_password }}"
+    loop:
+      - rocky.conf
+      - boefjes.conf
+      - mula.conf
+   
+  # Configure hostname in Rocky The DJANGO_ALLOWED_HOSTS and
+  # DJANGO_CSRF_TRUSTED_ORIGINS variables in /etc/kat/rocky.conf need to be
+  # configured with the hostname (or hostnames separated by commas) that will
+  # be used to access OpenKAT. If openkat.example.org is used to access OpenKAT
+  # the configuration should be:
+  # DJANGO_ALLOWED_HOSTS="openkat.example.org"
+  # TODO read IPv4 and IPv6 dict of IP's and add them all here separated with a comma.
+  #
+  - name: Configure hostname in Rocky. Currently we configure IP address. Please add FQDN later to  DJANGO_ALLOWED_HOSTS setting in /etc/kat/rocky.conf !
+    lineinfile:
+      path: "/etc/kat/rocky.conf"
+      regexp: "DJANGO_ALLOWED_HOSTS=.*"
+      line:  "DJANGO_ALLOWED_HOSTS=\"{{ ansible_default_ipv4.address }}\""
+  # DJANGO_CSRF_TRUSTED_ORIGINS="https://openkat.example.org"
+  # TODO read IPv4 and IPv6 dict of IP's and add them all here separated with a comma.
+  - name: Prepare DJANGO_CSRF_TRUSTED_ORIGINS setting
+    set_fact:
+      django_csrf: "https://{{ ansible_default_ipv4.address }}"
+  - name: Configure hostname in Rocky. Currently we configure IP address. Please add FQDN later to  DJANGO_CSRF_TRUSTED_ORIGINS setting in /etc/kat/rocky.conf !
+    lineinfile:
+      path: "/etc/kat/rocky.conf"
+      regexp: "DJANGO_CSRF_TRUSTED_ORIGINS=.*"
+      line:  "DJANGO_CSRF_TRUSTED_ORIGINS=\"{{ django_csrf }}\""
+
+  # Enable debugging on rocky
+  # default: DEBUG="False"
+    lineinfile:
+      path: /etc/kat/rocky.conf
+      regexp: "DEBUG=.*"
+      line:  "DEBUG=\"True\""
```

Comment:
DEBUG should only be used for development environments and maybe to diagnose problems, but shouldn't be turned on by default.